pipeline-model-definition/src/test/resources/whenBranchFalse.groovy

```diff
@@ -24,6 +24,9 @@
 
 pipeline {
     agent any
+    environment {
+        BRANCH_NAME = "master"
```

Comment:
Really? Is the environment not populated with the "from scm" settings that we use in the tests?